pkg/http/rest/api_test.go

```diff
@@ -0,0 +1,81 @@
+package rest
+
+import (
+	"github.com/TurnsCoffeeIntoScripts/jira-api-resource/pkg/configuration"
+	"net/http"
+	"testing"
+)
+
+var (
+	emptyValue           = ""
+	fakeUrl              = "https://github.com/TurnsCoffeeIntoScripts/jira-api-resource"
+	fakeUsername         = "dummy_username"
+	fakePassword         = "dummy_password"
+	fakeCustomFieldValue = "dummyValue"
+)
+
+type JiraAPIMock struct {
+}
+
+type DummyJsonObject struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [type DummyJsonObject should be DummyJSONObject](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)